pkg/blob/controllerserver.go

```diff
@@ -691,11 +693,17 @@ func (d *Driver) CreateBlobContainer(ctx context.Context, subsID, resourceGroupN
 			if getErr != nil {
 				return true, getErr
 			}
+			metadata := make(map[string]string)
+			metadata[createdByMetadata] = d.Name
+			container.Metadata = metadata
 			_, err = container.CreateIfNotExists(&azstorage.CreateContainerOptions{Access: azstorage.ContainerAccessTypePrivate})
 		} else {
+			metadataPtr := make(map[string]*string)
+			metadataPtr[createdByMetadata] = to.Ptr(d.Name)
 			blobContainer := armstorage.BlobContainer{
 				ContainerProperties: &armstorage.ContainerProperties{
 					PublicAccess: to.Ptr(armstorage.PublicAccessNone),
+					Metadata:     metadataPtr,
```

Comment:
Metadata: map[string]*string{createdByMetadata: to.Ptr(d.Name)}

Reply:
done